en_ID/navigation.md

```diff
@@ -50,7 +50,13 @@
   * [<img src="pages/images/twitter.png" alt="alt text" width="20px"/> Twitter](https://twitter.com/kulkultech)
   * [<img src="pages/images/youtube.png" alt="alt text" width="20px"/> Youtube](https://www.youtube.com/channel/UCkafa38JOKqTfZmBaMYoywQ)
 
-[gimmick:ForkMeOnGitHub (position: 'right', color: 'darkblue') ](https://github.com/kulkultech/open-source)
+<!--
```

Comment:
Why you change this?

Reply:
> Why you change this?

I apologise I have been making changes in the part you mentioned below. I have undone those changes. So now the logo should be visible on the page (including contrast and alignment change.